docs/tutorials/data_pipeline.md

```diff
@@ -146,28 +146,29 @@ For each operation, we list the related dict fields that are added/updated/remov
 
 ## Extend and use custom pipelines
 
-1. Write a new pipeline in any file, e.g., `my_pipeline.py`. It takes a dict as input and return a dict.
+1. Write a new pipeline e.g., `my_pipeline.py`. It takes a dict as input and returns a dict.
 
     ```python
+    import random
     from mmdet.datasets import PIPELINES
 
     @PIPELINES.register_module()
     class MyTransform:
+        def __init__(self, p=.5):
+            self.p = p
 
         def __call__(self, results):
-            results['dummy'] = True
+            if random.random() > self.p:
+                results['dummy'] = True
             return results
     ```
 
-2. Import the new class.
+2. Import and use the pipeline in your config file. 
+   Make sure the import is relative to where your train script is located.
 
     ```python
-    from .my_pipeline import MyTransform
-    ```
-
-3. Use it in config files.
+    custom_imports = dict(imports=['my_pipeline'], allow_failed_imports=False)
```

Comment:
maybe can add an annotation, `the path to my_pipeline, e.g. mmdet.datasets.pipelines.my_pipeline`